pkg/graph/schema.resolvers.go

```diff
@@ -630,6 +630,30 @@ func (r *queryResolver) Systems(ctx context.Context, after *string, first int) (
 	return conn, nil
 }
 
+func (r *systemIntakeResolver) Actions(ctx context.Context, obj *models.SystemIntake) ([]*model.SystemIntakeAction, error) {
+	actions, actionsErr := r.store.GetActionsByRequestID(ctx, obj.ID)
+	if actionsErr != nil {
+		return nil, actionsErr
+	}
+
+	var results []*model.SystemIntakeAction
+	for _, action := range actions {
+		graphAction := model.SystemIntakeAction{
+			ID:   action.ID,
+			Type: model.SystemIntakeActionType(action.ActionType),
+			Actor: &model.SystemIntakeActionActor{
+				Name:  action.ActorName,
+				Email: action.ActorEmail,
+				Eua:   action.ActorEUAUserID,
+			},
+			Feedback:  action.Feedback.Ptr(),
+			CreatedAt: *action.CreatedAt,
+		}
```

Comment:
Is this where you'd also attach the parent `SystemIntake`, or is this the circular reference cut-out?

Reply:
I'm not sure if we need to do that or if gqlgen will automatically call the existing resolver for system intakes instead.

When I try to run the following query against the server, I get an authorization error in the app console: `user unauthorized as owning the system intake	{"traceID": "5956a3bd-ae70-4064-b6a5-284742d53d69", "Authorized": false}`.

```gql
query {
  systemIntake(id: "ef86e03e-16ea-4757-92c7-971c42f028e5"){
    id
    actions {
      id
      systemIntake {
        id
      }
    }
  }
}
```

I'll try to dig in and figure out what is supposed to happen here next week.